help/destinations/catalog/advertising/liveramp-distribution.md

```diff
@@ -110,31 +112,60 @@ To configure details for the destination, fill in the required and optional fiel
 >title="Client name"
 >abstract="Your advertiser account name, as you would like it to be shown to the destination partner. Use your company name. Do not use spaces or special characters."
 
-### [!DNL Amobee] {#amobee}
+To configure details for the destination, fill in the fields below.
+
+![Platform UI image showing the customer data fields for the Acast destination.](../../assets/catalog/advertising/liveramp-distribution/LR_Acast_DestSpecific.png)
+
+* **[!UICONTROL Client name]**: Your advertiser account name, as you would like it to be shown to the destination partner. Use your company name. Do not use spaces or special characters.
 
 ### [!DNL Ampersand.tv] {#ampersand-tv}
 
+>[!CONTEXTUALHELP]
+>id="company_name"
```

Comment:
Line 124: This ID does not follow the correct format. Please update this to:

>id="platform_destinations_liveramp_distribution_ampersand_company_name"
>title="Your company name"
>abstract="Your company name, as you would like it to be shown to the destination partner. Do not use spaces or special characters."

Please update this tooltip ID in the destination config, too. It's using the wrong format:


 "tooltip": {
            "id": "company_name",
            "fallbackUrl": "https://www.adobe.com/go/destinationsliveramp-en"
          }

Also, unrelated to the docs, the **fallbackUrl** in the destination config seems to have been altered. The correct link is  https://www.adobe.com/go/destinations-liveramp-distribution-en